tmt.spec

```diff
@@ -115,6 +115,14 @@ Requires:       python3-mrack-beaker
 
 %description -n tmt+provision-beaker %_metapackage_description
 
+%package -n     tmt+provision-mock
+Summary:        Dependencies required for tmt mock provisioner
+Provides:       tmt-provision-mock == %{version}-%{release}
+Requires:       tmt == %{version}-%{release}
+Requires:       mock
+
+%description -n tmt+provision-mock %_metapackage_description
+
 # Replace with pyproject_extras_subpkg at some point
 %package -n     tmt+all
```

Comment:
This package (`tmt+all`) is missing `Requires:       tmt+provision-mock == %{version}-%{release}`

Reply:
Should it be a `Requires` though?

Reply:
Right, `tmt+all`, makes sense.